guides/doc-Release_Notes/master.adoc

```diff
@@ -15,6 +15,9 @@ endif::[]
 // Start inserting specific x.y.z releases here
 include::topics/foreman-3.6.1.adoc[leveloffset=+1]
 include::topics/foreman-3.6.0.adoc[leveloffset=+1]
+ifdef::katello[]
```

Comment:
We try to keep it chronological order (and perhaps Foreman needs to include dates too), so:
* F 3.6.1
* K 4.8.0
* F 3.6.0